demisto_sdk/commands/doc_reviewer/tests/rn_checker_test.py

```diff
@@ -82,6 +82,13 @@
             True,
         ),
         (["##### New: script", "Blah.", "##### script", "Blah."], False),
+        (
+            [
+                "#### Indicator Fields",
+                "- New: **Indicator Field**",
```

Comment:
Is the prefix dash needed here? I thought the fix was removing it from `rn_checker.py`